securedrop/tests/functional/functional_test.py

```diff
@@ -43,6 +47,44 @@ def __call__(self, driver):
             return True
 
 
+@contextmanager
+def capturel10n(app, what):
```

Comment:
I have not nearly enough idea what's happening here. Why are you `yield/return`-ing? Can you use longer variable names and function names?

Reply:
It is necessary to yield so the mocks do not go out of scope. They need to exist for the duration of the context. Is there a better way to do this?

Reply:
`yield`-ing is fine, but I've never seen the `yield/return` pattern like how you did. Can you explain that?

Reply:
Oh, that... well, it's a way of doing nothing if the environment variable is not set. Very hackish I don't like it.